gobblin-compaction/src/main/java/org/apache/gobblin/compaction/suite/CompactionSuiteBase.java

```diff
@@ -46,14 +50,17 @@
 @Slf4j
 public class CompactionSuiteBase implements CompactionSuite<FileSystemDataset> {
   public static final String SERIALIZE_COMPACTION_FILE_PATH_NAME = "compaction-file-path-name";
```

Comment:
Consider to remove this SERIALIZE_COMPACTION_FILE_PATH_NAME?

Reply:
+1